src/main/scala/mesosphere/marathon/event/http/HttpEventModule.scala

```diff
@@ -20,7 +20,7 @@ import scala.language.postfixOps
 trait HttpEventConfiguration extends ScallopConf {
 
   lazy val httpEventEndpoints = opt[String]("http_endpoints",
-    descr = "The URLs of the event endpoints",
+    descr = "The URLs of the event endpoints.",
```

Comment:
"The URLs of the event endpoints added to the current list of subscribers on startup. You can manage this list during runtime by using the `/v2/eventSubscriptions` API endpoint."